### PR TITLE
Dataset view part2

### PR DESCRIPTION
**Pull request by xavijam**
- Move notifier component (https://github.com/CartoDB/cartodb/pull/8321).
- Added notifier.
- Rename table.
- Rename table confirmation.
- Add menu bar.
- Change browser title when name changes.
- Disable (readonly) for users without table permission.
- Fix css problems.
- Disable state.
- First tests.

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/inline-editor/inline-editor-view.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ module.exports = CoreView.extend({
     }
 
     if (e.which === $.ui.keyCode.ENTER) {
-      this._onEdit && this._onEdit();
+      this._onEdit && this._onEdit(this.getValue());
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/inline-editor/inline-editor-view.js]: @nobuti, I've realized that it could be useful...

**Comment by xavijam** [comment]
Could you take a look @nobuti?

File: app/assets/stylesheets/editor-3/table.css.scss [hunk 2]
```diff
@@ -224,6 +232,10 @@ $tableBorderColor: $cSecondaryLine;
   z-index: 10;
   box-sizing: border-box;
 }
+.Table-paginator--relative {
+  position: absolute;
+  left: calc(50% - 71px);
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/table.css.scss]: I'd avoid magic numbers, maybe something like:
```
left: 50%;
transform: translateX(-50%);
```
should be better (if possible ofc)

File: lib/assets/javascripts/cartodb3/data/table-model.js [hunk 3]
```diff
@@ -49,6 +57,29 @@ module.exports = Backbone.Model.extend({
...
+    if (!userModel || !this._permissionModel) {
+      return false;
+    }
+    return this._permissionModel.isOwner(userModel);
+  },
+
+  getUnqualifiedName: function () {
+    var name = this.get('name');
+    if (!name) return null;
+    var tk = name.split('.');
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/table-model.js]: `"name.".split('.') --- ["name", ""]`

**Comment by nobuti** [comment]
👍

File: lib/assets/javascripts/cartodb3/data/table-model.js (hunk 3 above)
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/table-model.js]: That's not possible, there will be only a whatever.whatever when it belongs to other user, so for example:
`cdb.table_name`

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/data/table-model.js]: :adidas:

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/data/table-model.js]: :hummel:

**Comment by xavijam** [comment]
Merging due to previous frontend tests passed, and I've only changed a css property so, 🚀

**Pull request merged**